html/semantics/document-metadata/the-link-element/link-multiple-error-events.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<link rel="author" title="Dom Farolino" href="mailto:dom@chromium.org">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#the-link-element">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link id=link rel=stylesheet
+      onload="t.unreached_func('Sheet should fail to load')">
+<script>
+  const t = async_test("Check if the <link>'s error event fires for each style " +
+                     "sheet it fails to load");
+
+  link.onerror = t.step_func(() => {
```

Comment:
I think we try to avoid relying on the global scope polluter in wpt generally.

Reply:
OK, would a single page test be more suitable? Honestly I am not very familiar with them, so I am not sure if that is a solution to your comment.

Reply:
Oof nevermind I guess I can just write `async_test(t => {...})`...

Reply:
The global scope polluter comment refers to using (effectively) `window["link"]` for an element that has an id of `link`. It's easier to reason about tests that use explicit document.querySelector calls instead.

Reply:
Thanks, I didn't know that actually had a name. Fixed (+ went with `async_test(t => {})` anyways)

Reply:
Sorry for using unclear jargon :)